### ISSUE TITLE
Tick label is cropped when using Chinese font.

### ISSUE THREAD

**Issue by AllenDang** [gonum/plot#573]
### What are you trying to do?
Set the plot.DefualtFont to a Chinese font and tick in x-axis should be displayed completely.

### What did you do?
Set font to /System/Library/Fonts/STHeiti Light.ttc
```
        fontData, err := ioutil.ReadFile(fontPath)
	if err != nil {
		panic(err)
	}

	font, err := truetype.Parse(fontData)
	if err != nil {
		panic(err)
	}

	fontName := "font"
	vg.AddFont(fontName, font)
	plot.DefaultFont = fontName
	plotter.DefaultFont = fontName
```

### What did you expect to happen?
Tick should be displayed completely.
![截屏2020-01-09上午10 30 58](https://user-images.githubusercontent.com/18550/72032924-c39fc880-32cb-11ea-939d-0af0bb8e2365.png)

### What actually happened?
Tick label is cropped.
![截屏2020-01-09上午10 27 29](https://user-images.githubusercontent.com/18550/72032945-d0242100-32cb-11ea-89cc-fb4d1f7c02c7.png)

### What version of Go and Gonum/plot are you using?
go version go1.13.5 darwin/amd64
gonum/plog version: ca91d9d40d0a39380a55109c45269a2462d2b5b6

### Does this issue reproduce with the current master?
Yes

**Comment by kortschak** [comment]
Can you please provide a complete reproducer and link to the font file.

**Comment by AllenDang** [comment]
@kortschak 
Here is a minimal reproducer.
```
package main

import (
	"io/ioutil"
	"math/rand"

	"github.com/golang/freetype/truetype"
	"gonum.org/v1/plot"
	"gonum.org/v1/plot/plotter"
	"gonum.org/v1/plot/vg"
)

func main() {
	fontPath := "/System/Library/Fonts/STHeiti Light.ttc"
	fontData, err := ioutil.ReadFile(fontPath)
	if err != nil {
		panic(err)
	}

	font, err := truetype.Parse(fontData)
	if err != nil {
		panic(err)
	}

	fontName := "font"
	vg.AddFont(fontName, font)
	plot.DefaultFont = fontName
	plotter.DefaultFont = fontName

	data := make(plotter.XYs, 10)
	for i := range data {
		data[i].X = float64(i)
		data[i].Y = float64(rand.Intn(10))
	}
	p, _ := plot.New()

	l, _ := plotter.NewLine(data)

	p.Add(l)

	p.Save(400, 200, "output.png")
}
```
Here is the font link.
[STHeiti](https://github.com/potyt/fonts/blob/master/macfonts/STHeiti/ttc/STHeiti%20Light.ttc)

**Comment by sbinet** [comment]
this was fixed w/ a51a196
(ie: https://github.com/gonum/plot/pull/642)
